.github/workflows/integration-tests-on-release.yaml

```diff
@@ -18,34 +18,43 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions : ['v1.15.12','v1.18.15','v1.21.0']
+        kubernetes-versions : ['v1.15.12','v1.16.15','v1.18.15','v1.19.7','v1.20.2']
     steps:
     - name: checkout
       uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
 
     - name: setup golang
       uses: actions/setup-go@v2
       with:
         go-version: 1.16
 
     - name: setup minikube
-      uses: manusa/actions-setup-minikube@v2.4.2
+      uses: manusa/actions-setup-minikube@v2.3.1
       with:
-        minikube version: 'v1.21.0'
+        minikube version: 'v1.13.1'
         kubernetes version: ${{ matrix.kubernetes-versions }}
         start args: --memory 6g --cpus=2
         github token: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: print k8s cluster status
-      run: tests/scripts/github-action-helper.sh print_k8s_cluster_status
+    - name: check k8s cluster status
+      run: |
+        kubectl cluster-info
+        kubectl get pods -n kube-system
 
     - name: use local disk
-      run: tests/scripts/github-action-helper.sh use_local_disk_for_integration_test
+      run: |
+        sudo swapoff --all --verbose
+        sudo umount /mnt
+        # search for the device since it keeps changing between sda and sdb
+        sudo wipefs --all --force /dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1
+        sudo lsblk
 
     - name: build rook
-      run: tests/scripts/github-action-helper.sh build_rook
```

Comment:
So instead of backporting the script, we are just copying the contents of the function. Why not just backport the script?

Reply:
On my privious PR I copy the changes of `integration-test-on-release.yaml` form master which was wrong. So. now I took the changes from what it had in release-1.6 branch. Same for all the comments.

And, the script getting modified by the time and also new changes were added so I just made changes according to release-1.6.

Reply:
as discussed offline I just copied the changes directly from release-1.7 branch